### PR TITLE
Registration Page

### PR DESCRIPTION
**Pull request by apappu9**

### CODE REVIEW

File: src/data/states.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "Massachusetts",
+  "Maryland",
+  "Maine",
+  "Michigan",
+  "Minnesota",
+  "Missouri",
+  "Mississippi",
+  "Montana",
+  "North Carolina",
+  " North Dakota",
```
Comment (brianstrauch) [inline src/data/states.json:32]: ope

File: src/scenes/Registration/sections/RaceDemographics/index.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const raceOptions = [
+  { label: 'American Indian or Alaskan Native', value: 'American Indian or Alaskan Native' },
+  { label: 'Asian', value: 'Asian' },
+  { label: 'Black or African American', value: 'Black or African American' },
+  { label: 'Native Hawaiian or Pacific Islander', value: 'Native Hawaiian or Pacific Islander' },
+  { label: 'Hispanic or Latino or Spanish Origin', value: 'Hispanic or Latino or Spanish Origin' },
+  { label: 'White', value: 'White' },
+  { label: 'Other', value: 'Other', isOther: true },
+  { label: 'I’m optional! :) (Prefer not to answer)', value: 'Prefer Not to Say' },
```
Comment (brianstrauch) [inline src/scenes/Registration/sections/RaceDemographics/index.jsx:14]: This phrasing is weird. What does "I'm optional" mean?

File: src/scenes/Registration/sections/RaceDemographics/index.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      <div className={styles.note}>
+        <p>
+          <i><b>Note:</b> This is a purely optional question! :) In light of recent events, we’re collecting this
+          information in order to determine what our current attendee demographics are and how we can work on
+          increasing diversity! Don’t forget that it’s purely optional and won’t affect anything!</i>
+        </p>
+      </div>
+    </div>
+
+    <Buttons className={styles.buttons} />
```
Comment (brianstrauch) [inline src/scenes/Registration/sections/RaceDemographics/index.jsx:37]: These buttons need some margin under them, they're touching the bottom of the screen.

File: src/scenes/Registration/sections/Education/index.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  .concat('Other')
+  .concat('N/A')
+  .map(major => ({ value: major, label: major }));
+
+
+const Education = ({ Buttons }) => (
+  <section className={styles.education}>
+    <h1 className={styles.title}>Education</h1>
+    <div className={styles.content}>
+      <SelectField name="degreePursued" options={degreeOptions} placeholder="What degree are you currently pursuring? *" />
```
Comment (brianstrauch) [inline src/scenes/Registration/sections/Education/index.jsx:38]: typo: "pursuring"

File: src/scenes/Registration/sections/Experience/index.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+const programmingYearsOptions = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+  .map(years => ({ value: years, label: years === 10 ? '10+' : `${years}`}));
+
+const abilityOptions = [
+  { value: 1, label: '1 - What is code?' },
+  { value: 2, label: '2 - I am familiar with code, but need guidance. ' },
+  { value: 3, label: '3 - I am comfortable working on independent projects.' },
+  { value: 4, label: '4 - I am comfortable writing and reviewing production level code.' },
+  { value: 5, label: '5 - I am code.' },
```
Comment (brianstrauch) [inline src/scenes/Registration/sections/Experience/index.jsx:15]: Nice.

File: src/scenes/Registration/sections/RaceDemographics/index.jsx (hunk 2 above)
Comment (PatrickKan) [reply src/scenes/Registration/sections/RaceDemographics/index.jsx:14]: :( I thought it was a cute way to ask the race question in a less dramatic way

Comment (PatrickKan) [reply src/scenes/Registration/sections/RaceDemographics/index.jsx:14]: Jk, I think just "Optional! :) (Prefer not to answer)" would be better

File: src/scenes/Registration/sections/RaceDemographics/index.jsx (hunk 3 above)
Comment (apappu9) [reply src/scenes/Registration/sections/RaceDemographics/index.jsx:37]: I think I fixed this, but can you check again? If it's not, what screen size are you on?

**Comment by apappu9** [comment]
Also @brianstrauch, the application will now let you identify as the linter you were always meant to be :) (the feature is somewhat hidden though)

**Comment by brianstrauch** [comment]
> Also @brianstrauch, the application will now let you identify as the linter you were always meant to be :) (the feature is somewhat hidden though)

i mean it’s 2020, it’s about time my gender was publicly recognized

**Pull request merged**